nemo/collections/asr/parts/preprocessing/segment.py

```diff
@@ -261,22 +262,38 @@ def segment_from_file(
 
         Note that audio_file can be either the file path, or a file-like object.
         """
+        is_segmented = False
```

Comment:
This new functionality needs some tests in https://github.com/NVIDIA/NeMo/blob/main/tests/collections/asr/test_preprocessing_segment.py